.travis.yml

```diff
@@ -201,9 +206,17 @@ before_cache:
   # Make the cache stable between builds by removing build output
   - rm -rf $HOME/.m2/repository/io/prestosql
 
-notifications:
-  slack:
-    secure: NPHH9nIuInY3v4Jvl1ywC5BlygY9YiVni6LLcR52nwdx/FocO9j80aNKx/kuso+nGRM3dmWKPWdk8ShOOS2ZRvYXQUe2Gy6wSnlwqjKGaHEReZA5hOFuYfTQw0Gh7WGzwiuYpvAzhXzn31GvL92M24Oq0kqP64sA9nndBAT3uMv5PdQV40JtL8kQdh66wHuwtJ/LrljWKTwrTJH8NcGc7I6MLljxe+3n1zy49J7MqVyGdBdFA6CMc6Abgc3XKfsadJH3QFC05LfyOQqMjCutSd+HWkl7yMFLWpddLj0C9DdpG1z7324gcQjlLbDGI3Mc3vd/bTLnanIgQ4ZIWSK8/0j8u/dJw0DPUkO82sZ+1nvtwNvE+C67M2vxlunRuxEwEIw7a6cUuguKh/EhZYDccF6523kFPyCLs8zSKdScCjjz4pj+jj3niug/tBiSRebwhcfLnWlQp7trY54ETtvwjAidGfXwuqO07FsNJL2gweg+axFhcSqFos7VwWeJVB1bVGg1PYM3wFarr25aQstIvE+Ap9Mym1MXXhswwe09BYQiOzoiK/CYAKrQT3IjLBGOM8P+uh2rv4w+wUNlgfUnG9A9aiRAlw+9uXDgFIR59y3SylKtWabDtX5khO/5TTlzc4Cq3tGSoNPPJ+WAtUISGrgCjQAVVnhdZ7chi4aLt5s=
+# Slack update to Criteo's internal boss-db channel
```

Comment:
we could just yank this out in a future commit (or redirect if you want to the new chan)